packages/L10n/src/locales/en.yml

```diff
@@ -7,25 +7,26 @@ en:
     dataGrid:
       a11yTextMessage: "{{value}}. You are on row {{rowIndex}}, column {{columnIndex}}. Disregard the following information:"
     uploader:
-      label: Drag and drop or choose files from your computer
-      overlay:
-        title: Drag and drop
-        subtitle: Your files here
-      form:
-        dragAndDrop: Drag and drop
-        or: or
-        chooseFiles: Choose files
-        from: From your computer
+      drop_files: Drop files to upload
+      drop_files_here_or: Drop files to upload here or
+      choose_from_computer: choose from your computer
+      cancel_upload: Cancel upload
+      restart_upload: Restart upload
       errors:
-        file_size: exceeds the maximum {{size}} file size.
-        file_type: "{{type}} is incorrect file type."
+        size: File must be smaller than {{maxFileSize}}
+        type: "File must be one of the following types: {{supportedTypes}}"
```

Comment:
I see `&#x2F;` in the UI...

Reply:
<img width="485" alt="Screen Shot 2020-05-28 at 3 06 50 PM" src="https://user-images.githubusercontent.com/23224777/83199477-1bfa2a00-a0f6-11ea-9d3c-c9187a86b7a5.png">


@mikrotron Fixed